lib/plistLib/main.js

```diff
@@ -250,8 +252,8 @@ define([
             dict: function(doc, dict) {
                 var fragment = doc.createDocumentFragment(),
                     key, value, keyElement;
-                for(key in dict) { 
-                    if(!dict.hasOwnProperty(key) || dict[key] === undefined || dict[key] === null)
+                for(key in dict) {
+                    if(!Object.prototype.hasOwnProperty.call(dict, key) || dict[key] === undefined || dict[key] === null)
```

Comment:
again quite interested in why this was changed, as the former is the "correct" JS pattern. What made it not work?